packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.tsx

```diff
@@ -12,14 +12,15 @@ import {
     FASTSelect,
     FASTTextField,
 } from "@microsoft/fast-components";
-
+import { FASTColorPicker } from "./css/color-picker/";
```

Comment:
We generally leave off the trailing slash.

```suggestion
import { FASTColorPicker } from "./css/color-picker";
```

Reply:
Fixed.